Description: Fix to rebuild pdf files
Author: Alastair McKinstry <mckinstry@debian.org>
Last-Updated: 2018-03-12
Forwarded: no

 doc: docs/qd.pdf
Index: qd-2.3.20+dfsg.1/Makefile.am
===================================================================
--- qd-2.3.20+dfsg.1.orig/Makefile.am
+++ qd-2.3.20+dfsg.1/Makefile.am
@@ -23,7 +23,7 @@ dist_noinst_DATA = docs/Makefile \
 doc: docs/qd.pdf
 
 docs/qd.pdf:
-	$(MAKE) -C docsqd.pdf
+	$(MAKE) -C docs qd.pdf
 
 changelog:
 	git log >ChangeLog
